packages/cli/src/cog/job.factory.ts

```diff
@@ -22,7 +22,9 @@ export const CogJobFactory = {
    */
   async create(ctx: JobCreationContext): Promise<CogJob> {
     const id = ctx.override?.id ?? ulid.ulid();
-    const imageryName = basename(ctx.sourceLocation.path).replace(/\./g, '-'); // batch does not allow '.' in names
+    let imageryName = ctx.imageryName;
+    if (imageryName == null) imageryName = basename(ctx.sourceLocation.path).replace(/\./g, '-'); // batch does not allow '.' in names
```

Comment:
this was removed, we dont use the imagery name inside of batch so we can remove the .replace

Reply:
Fixed. Thanks.